app/components/funding_widget_component.html.erb

```diff
@@ -58,7 +58,7 @@
           <div>
             <h3>Next steps</h3>
             <div>
-            Find information around your eligibility and how to apply on the page below.
+            Continue on this page to find out more about your eligibility and how to receive your scholarship or bursary.
```

Comment:
Not sure this quite works in practice @Dom-Collins as this message is now appearing on subjects that are bursary only? 

![image](https://user-images.githubusercontent.com/87642394/195871920-3909c206-7d15-4adf-974a-7a14671db03d.png)

Reply:
Hi @gemmadallmandfe How about something more generic like: 

"Continue on this page to find out if you're eligible for a scholarship or bursary."

or

"Continue on this page to find out more about your eligibility for a scholarship or bursary."

What do you think @EllieNodder ?

Reply:
I agree - I'd suggest keeping it a bit more generic - maybe

'Continue on this page to find out more about your eligibility for funding.'